### PR TITLE
feat: user openapi spec

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

* finishing users tag openapi spec
* added missing descriptions to the Frontend SDK endpoints so that meta rules are satisfied

### CODE REVIEW

File: src/lib/openapi/spec/profile-schema.ts [hunk 1]
```diff
@@ -6,22 +6,29 @@ export const profileSchema = {
     $id: '#/components/schemas/profileSchema',
     type: 'object',
     additionalProperties: false,
+    description: 'User profile overview',
     required: ['rootRole', 'projects', 'features'],
     properties: {
         rootRole: {
+            description:
```
Comment (thomasheartman) [inline src/lib/openapi/spec/profile-schema.ts:13]: the same note about $refs goes here. I might be wrong though, but I think we checked this?

File: src/lib/openapi/spec/proxy-client-schema.ts [hunk 2]
```diff
@@ -22,6 +23,8 @@ export const proxyClientSchema = {
         environment: {
             type: 'string',
             deprecated: true,
+            description: 'deprecated',
+            example: 'deprecated',
```
Comment (thomasheartman) [inline src/lib/openapi/spec/proxy-client-schema.ts:27]: Could probably use a standard env name here, but `deprecated` works too. Your call.
```suggestion
            example: 'development',
```

**Review by thomasheartman — APPROVED** [review]
Looks good! A couple minor suggestions, but nothing major 🔥

File: src/lib/openapi/spec/feedback-schema.ts [hunk 3]
```diff
@@ -4,20 +4,29 @@ export const feedbackSchema = {
     $id: '#/components/schemas/feedbackSchema',
     type: 'object',
     additionalProperties: false,
+    description: 'User feedback information',
     properties: {
         userId: {
+            description: 'Identifier of the current user giving feedback',
             type: 'number',
+            example: 2,
```
[the remark this answers is not in the document]
Comment (kwasniew) [reply src/lib/openapi/spec/feedback-schema.ts:12]: I think so

File: src/lib/openapi/spec/feedback-schema.ts [hunk 4]
```diff
@@ -4,20 +4,29 @@ export const feedbackSchema = {
...
         },
         feedbackId: {
+            description: 'The name of the feedback session',
             type: 'string',
+            example: 'pnps',
         },
         neverShow: {
+            description: 'Whether this feedback should be disabled',
```
[the remark this answers is not in the document]
Comment (kwasniew) [reply src/lib/openapi/spec/feedback-schema.ts:20]: Added more precise description: `true` when user opts-in to never show the feedback again. This is UI only indicator to stop getting feedback prompt.

**Pull request merged**